### PR TITLE
adding content of uninstalling metering

### PR DESCRIPTION
**Pull request by lamek**

### CODE REVIEW

**Comment by lamek** [comment]
@pruan-rht appreciate feedback. This is a new topic for uninstalling metering. I also moved the existing line about cleaning up S3 buckets into a note.

File: modules/metering-uninstall.adoc [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+To remove metering:
+
+. In the {product-title} web console, click *Operators* -> *Installed Operators*.
+
+. Find the metering operator and click the {kebab} menu. Select *Uninstall Operator*.
+
+. A dialog opens. Click *Remove* to uninstall metering.
+
+[NOTE]
+====
```
Comment (pruan-rht) [inline modules/metering-uninstall.adoc]: @lamek this section is duplicated in `modules/metering-store-data-in-s3.adoc` ...unless that's intended.

Comment (lamek) [reply modules/metering-uninstall.adoc]: I left that intentionally. I think the behavior is unintuitive and therefore good to call out where customers are most likely to see it. Unless your strongly opposed, I vote we keep this in both locations.

Comment (pruan-rht) [reply modules/metering-uninstall.adoc]: @lamek ok that's fine....the rest lgtm

File: modules/metering-store-data-in-s3.adoc [hunk 2]
```diff
@@ -5,7 +5,12 @@
 [id="metering-store-data-in-s3_{context}"]
 = Storing data in Amazon S3
 
-Metering can use an existing Amazon S3 bucket or create a bucket for storage. Metering does not manage or delete any S3 bucket data. When uninstalling metering, any S3 buckets used to store metering data must be manually cleaned up.
+Metering can use an existing Amazon S3 bucket or create a bucket for storage.
```
Comment (mburke5678) [inline modules/metering-store-data-in-s3.adoc]: Did you mean _Metering can... create a bucket for storage._  Or _you can create a bucket for storage_?

File: modules/metering-uninstall.adoc [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+* Metering must be installed.
+
+.Procedure
+
+To remove metering:
+
+. In the {product-title} web console, click *Operators* -> *Installed Operators*.
+
+. Find the metering operator and click the {kebab} menu. Select *Uninstall Operator*.
```
Comment (mburke5678) [inline modules/metering-uninstall.adoc]: s/metering/Metering ??

File: modules/metering-store-data-in-s3.adoc (hunk 2 above)
Comment (lamek) [reply modules/metering-store-data-in-s3.adoc]: The odd part (to me), is that you have to manually specify where you want to use storage. As part of that, you have the option to use an existing bucket or tell metering to create a new bucket when you apply your config.

File: modules/metering-uninstall.adoc [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+
+. A dialog opens. Click *Remove* to uninstall metering.
```
[the remark this answers is not in the document]
Comment (lamek) [reply modules/metering-uninstall.adoc]: I can change to, "In the dialog box, click *Remove* to uninstall metering."

**Pull request merged**